timesketch/lib/tasks.py

```diff
@@ -519,3 +522,40 @@ def run_csv_jsonl(file_path, events, timeline_name, index_name, source_type):
     _set_timeline_status(index_name, status='ready')
 
     return index_name
+
+
+@celery.task(track_started=True, base=SqlAlchemyTask)
+def run_mans(file_path, events, timeline_name, index_name, source_type):
+    """Create a Celery task for processing mans file.
+
+    Args:
+        file_path: Path to the mans file.
+        events: A string with the events. Not used in mans.
+        timeline_name: Name of the Timesketch timeline.
+        index_name: Name of the datastore index.
+        source_type: Type of file, csv or jsonl.
+
+    Returns:
+        Name (str) of the index.
+    """
+    # Log information to Celery
+    message = 'Index timeline [{0:s}] to index [{1:s}] (source: {2:s})'
+    logging.info(message.format(timeline_name, index_name, source_type))
+
+    elastic_host = current_app.config['ELASTIC_HOST']
+    elastic_port = int(current_app.config['ELASTIC_PORT'])
+    try:
+        mte = MansToEs(filename=file_path, name=timeline_name, index=index_name,
+                       es_host=elastic_host, es_port=elastic_port)
+        mte.run()
+    except Exception as e:  # pylint: disable=broad-except
```

Comment:
is there no way to narrow down the exception handling here?

or have more specific ones and an "exception as e" as the last resort?

Reply:
There are many things that could go wrong: billiard, elastic, xmltodict so it will be difficult to proper manage it.
A solution similar to the one in present run_csv_jsonl could be ok?

Reply:
I was mainly wondering whether there are specific exceptions that you know about htat you can start catching and then doing a more proper error report, and then have this as a catch all if those aren't present, but others are...

except IOError as e:
  error_msg = 'Something something'...
except RuntimeError as e:
  ...

except Exception as e:
...

If not, then I'm fine with this, just asking the question

Reply:
Unfortunately not, so let's keep in this way.